configure.py

```diff
@@ -1492,6 +1492,13 @@ def main():
     else:
       set_trisycl_include_dir(environ_cp)
 
+  set_action_env_var(environ_cp, 'TF_NEED_ROCM', 'ROCm', False)
```

Comment:
`TF_NEED_CUDA`, `TF_NEED_SYCL` and `TF_NEED_ROCM` are all mutually exclusive, right? If so, we need a sanity check here that at most one of them is true.

Reply:
From TensorFlow perspective they are not mutually exclusive. It's possible to enable all three and still have TensorFlow built.

Reply:
ok will do.

Reply:
revised the PR to raise error in case more than 2 GPU platforms are specified (CUDA/SYCL/ROCm).